main/share-service-context.js

```diff
@@ -109,6 +110,12 @@ class ShareServiceContext {
 
     this.config.openInEditor();
   }
+
+  waitForDeepLink() {
```

Comment:
```suggestion
  async waitForDeepLink() {
```

For readability, I prefer to mark any function that returns a promise with `async`.